testcases/MachineConfig.py

```diff
@@ -494,6 +519,11 @@ def CecSetup(self):
             self.lmb_setup()
         if self.cec_dict['hugepages'] is not None:
             self.hugepage_16gb_setup()
+        if self.cec_dict['iocapacity'] is not None:
+            if bmc_type == "EBMC_PHYP":
+                self.io_enlarge_cpacity()
```

Comment:
also similarly we can use self.cv_BMC.io_enlarge_capacity() can be directly used

Reply:
Sure, WIll address this as suggested